src/api/spec/views/webui/user/show.html.erb_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
```

Comment:
NO!!! hahaha. Just testing that 'when the current user is nobody' it 'hides the email' and  'when the current user is valid'  it "displays the email",..., no matter the view code... just that works like that